framework_tvb/tvb/interfaces/web/controllers/users_controller.py

```diff
@@ -36,27 +36,29 @@
 .. moduleauthor:: Lia Domide <lia.domide@codemart.ro>
 """
 
-import os
 import json
-import time
+import os
 import ssl
+import time
+from urllib.request import urlopen
+
 import cherrypy
 import formencode
-from urllib.request import urlopen
+import tvb.interfaces.web
 from formencode import validators
-from tvb.core.entities.file.files_update_manager import FilesUpdateManager
-from tvb.core.services.user_service import UserService, KEY_PASSWORD, KEY_EMAIL, KEY_USERNAME, KEY_COMMENT
 from tvb.basic.profile import TvbProfile
-from tvb.core.services.project_service import ProjectService
+from tvb.core.entities.file.files_update_manager import FilesUpdateManager
 from tvb.core.services.exceptions import UsernameException
+from tvb.core.services.project_service import ProjectService
+from tvb.core.services.texture_to_json import color_texture_to_list
+from tvb.core.services.user_service import UserService, KEY_PASSWORD, KEY_EMAIL, KEY_USERNAME, KEY_COMMENT, \
+    KEY_AUTH_TOKEN
 from tvb.core.utils import format_bytes_human, hash_password
-import tvb.interfaces.web
+from tvb.interfaces.rest.server.security.authorization import AuthorizationManager
```

Comment:
This dependency of tvb.interface.web on tvb.interfaces.rest is odd.
We should avoid it. Can we move the AuthorizationManager down under tvb.core? Maybe even tvb.core.services?